src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -132,17 +133,15 @@ public override object VisitSqlRoot(SqlRootExpression expression, SearchOptions
                 StringBuilder.Append(expression.SearchParamTableExpressions.Count > 0 ? "CAST(IsMatch AS bit) AS IsMatch, " : "CAST(1 AS bit) AS IsMatch, ");
                 StringBuilder.Append(expression.SearchParamTableExpressions.Count > 0 ? "CAST(IsPartial AS bit) AS IsPartial, " : "CAST(0 AS bit) AS IsPartial, ");
 
-                if (_schemaInfo.Current > 3)
-                {
-                    // IsRawResourceMetaSet column was added in V4
-                    StringBuilder.Append(VLatest.Resource.IsRawResourceMetaSet, resourceTableAlias).Append(", ");
-                }
-                else
+                StringBuilder.Append(VLatest.Resource.IsRawResourceMetaSet, resourceTableAlias).Append(", ");
+
+                if (_schemaInfo.Current >= SchemaVersionConstants.SearchParameterHashSchemaVersion)
                 {
-                    StringBuilder.Append("CAST(0 AS bit) AS IsRawResourceMetaSet, ");
```

Comment:
>StringBuilder.Append("CAST(0 AS bit) AS IsRawResourceMetaSet, "); [](start = 20, length = 65)

Why we getting rid of this line?

Reply:
Good question, there are two things going on here. The first thing is I'm removing the check for the schema version being greater than or equal to V4, since V4 is now the minimum schema version.

The second thing is I am adding a check to see if the schema version is greater than or equal to V8, since the query will differ depending on if there is a column for the search parameter hash value in `dbo.Resource`.